ansible/requirements.yml

```diff
@@ -0,0 +1,19 @@
+---
+# Sorted by maintainer name, role name
+- src: mrlesmithjr.squid
+- src: oasis-roles.kdump
+- src: resmo.ntp
+- src: singleplatform-eng.users
+- src: https://github.com/stackhpc/ansible-role-cluster-infra
+  version: newton-backport
+  name: stackhpc.cluster-infra
+- src: https://github.com/cedadev/jasmin-cluster-infra-ansible
+  name: jasmin.cluster-infra
+- src: https://github.com/stackhpc/ansible-role-cluster-nfs
+  name: stackhpc.cluster-nfs
+- src: https://github.com/stackhpc/ansible-role-openhpc
+  name: stackhpc.openhpc
+  version: mark-verbal-feedback
```

Comment:
We'll need to change this to remove the `version`, it got merged and the `mark-verbal-feedback` branch has been deleted.

Reply:
Does this mean we should now pull directly from galaxy or should the source still be the GitHub repo?

Reply:
Aha, good point.  Probably yes.